subprojects/bootstrap/src/main/java/org/gradle/launcher/bootstrap/EntryPoint.java

```diff
@@ -75,6 +77,7 @@ public void onFailure(Throwable failure) {
             this.failure = failure;
         }
 
+        @CheckForNull
```

Comment:
Because the package is annotated `@NonNull` IJ complains about line 53:
https://github.com/gradle/gradle/pull/12675/files#diff-9a36e1cda08027f61523a5a9c964a7afR55